hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/file/tfile/TestTFile.java

```diff
@@ -109,15 +109,15 @@ private int readAndCheckbytes(Scanner scanner, int start, int n)
       byte[] val = readValue(scanner);
       String keyStr = String.format(localFormatter, i);
       String valStr = value + keyStr;
-      assertTrue("btyes for keys do not match " + keyStr + " "
+      assertTrue("bytes for keys do not match " + keyStr + " "
```

Comment:
pull this into a const and it could be referred everywhere.

Also, this is the time to move to assertEquals, if it does array comparisons. If not, then never mind